src/main/resources/templates/admin.html

```diff
@@ -12,61 +12,65 @@
             <div class="sightings-pending-container">
 
                 <h1>Pending Sightings</h1>
+
```

Comment:
It's personal preference, but I would advice against having quite so many new lines between all the elements in your html. I feel especially strongly that they are not needed in lines 68-76, where the indentation is already providing a clear distinction between the different elements